### PR TITLE
Fix DNS documents to remove references to DNS as an optional addon.

### PR DESCRIPTION
**Pull request by girishkalele**

### CODE REVIEW

**Comment by girishkalele** [comment]
@bprashanth Fix for #432 

(umbrella issue https://github.com/kubernetes/kubernetes/issues/9107)

File: docs/admin/dns.md [hunk 1]
```diff
@@ -15,25 +15,24 @@ in namespace `bar` can look up this service by simply doing a DNS query for
...
 ## Issues
 
-The skydns service is reachable directly from Kubernetes nodes (outside
-of any container) and DNS resolution works if the skydns service is targeted
-explicitly. However, nodes are not configured to use the cluster DNS service or
-to search the cluster's DNS domain by default.  This may be resolved at a later
-time.
+The 1.3 release introduced Federation (Ubernetes) support for multisite Kubernetes installations. There are
+DNS changes introduced that will allow the lookup of services using a six part DNS name.
+See [Federation docs](https://kubernetes.github.io/docs/admin/multiple-zones.md)
```
Comment (johndmulhausen) [inline docs/admin/dns.md]: URL should be /docs/admin/multiple-zones/

**Comment by girishkalele** [comment]
@thockin - Fixing old DNS docs

**Comment by johndmulhausen** [comment]
Thanks, @girishkalele

**Pull request merged**